app/javascript/components/server-components/DashboardPage.tsx

```diff
@@ -316,6 +316,7 @@ export const DashboardPage = ({
       <PageHeader
         title="Dashboard"
         actions={Object.keys(tax_forms).length > 0 && <DownloadTaxFormsPopover taxForms={tax_forms} />}
+        className="border-b-0 sm:border-b"
```

Comment:
Fine for now, but we should update this in the `PageHeader` component - we always hide the title on mobile so we can update to hide the whole thing on mobile if there's no actions/children